server.js

```diff
@@ -323,9 +325,27 @@ export async function startServer() {
   }
 
   const httpServer = app.listen(PORT, () => {
-    console.log(
-      `Server started at ${http2 ? "https" : "http"}://localhost:${PORT}`,
+    const scheme = http2 ? "https" : "http";
+    const url = `${scheme}://localhost:${PORT}`;
+    console.log(`Server started at ${url}`);
+    // Auto open browser
+    const OPEN_BROWSER_ON_START = JSON.parse(
```

Comment:
Let's move this into `./components/env`, and parse the environment variable like others.